cashu/core/split.py

```diff
@@ -1,8 +1,7 @@
 def amount_split(amount):
```

Comment:
As you're changing the code anyway it would be good add a type hint:
```suggestion
def amount_split(amount: int) -> List[int]:
```

Reply:
Good idea, done.